runtime/src/main/java/org/corfudb/protocols/wireprotocol/ICorfuPayload.java

```diff
@@ -397,8 +397,6 @@
             buffer.writeLong(streamRange.getEnd());
         } else if (payload instanceof Roaring64NavigableMap) {
             Roaring64NavigableMap mrb = (Roaring64NavigableMap) payload;
-            // Improve compression
-            mrb.runOptimize();
```

Comment:
Could you please help explain why we are removing runOptimize()?

Reply:
This code path is called when the sequencer replies to a clients stream query, which is usually a small message. I don't think optimizing the bitset just for transfer is worth the latency cost because the bitset is relatively small to begin with.